### PR TITLE
Mockup v3

### PR DESCRIPTION
**Pull request by marrts**
- Added new mockup and end effector meshes

- Added first scanned part mesh and its toolpaths

### CODE REVIEW

**Comment by jrgnicho** [comment]
Can you add instructions on how to run the motion planning node(s) with gazebo and rviz?

**Comment by marrts** [comment]
Added instructions and made some minor changes to help improve the freespace planning slightly

File: crs_motion_planning/src/freespace_planning_server.cpp [hunk 1]
```diff
@@ -222,7 +230,7 @@ class PlanningServer: public rclcpp::Node
     traj_pc->smooth_velocities = true;
     traj_pc->smooth_accelerations = true;
     traj_pc->smooth_jerks = true;
-    traj_pc->longest_valid_segment_length = 0.1;
+    traj_pc->longest_valid_segment_length = 0.05;
```
Comment (jrgnicho) [inline crs_motion_planning/src/freespace_planning_server.cpp:233]: Should this be made into a configurable parameter?

Comment (marrts) [reply crs_motion_planning/src/freespace_planning_server.cpp:233]: In the next iteration of the full process planner I plan to make it more configurable, but I think this freespace planner node is okay how it is now.

**Comment by jrgnicho** [comment]
@marrts 
I'm following your instructions but I'm getting the following gazebo related error
```
 Failed to load plugin libgazebo_ros_factory.so: libgazebo_ros_factory.so: cannot open shared object file: No such file or directory

```
Once I added 
```
  <depend>gazebo_ros</depend>  
  <depend>gazebo_plugins</depend>
  <depend>gazebo_ros_camera</depend>
```
 to the crs_support/package.xml and ran `rosdep ...` it loaded the required plugins  

But then it printed an error regarding a missing file
```
[Err] [InsertModelWidget.cc:405] Missing model.config for model "/home/jrgnicho/.gazebo/models/crs_support"
[gazebo-2] [Err] [REST.cc:205] Error in REST request
```

I also ran into another occasional error where there was an existing instance of the gazebo server,
I think we could add the following command to the launch file
```
killall -9 gazebo & killall -9 gzserver  & killall -9 gzclient
```
in order to kill any lingering instances of gazebo prior to running our application

**Comment by marrts** [comment]
I think adding that command to kill any lingering instances of gazebo makes sense. I also get the error that I am missing the model.config, but it doesn't prevent me from actually running the launch and gazebo coming up. It's my understanding that the model.config is unnecessary, but I will look into adding one. Is the gazebo world with the model coming up for you?

**Comment by jrgnicho** [comment]
> Is the gazebo world with the model coming up for you?

Yes it is

**Comment by jrgnicho** [comment]
The demo works well though

**Comment by jrgnicho** [comment]
@marrts will you be making any additional changes to this?

**Comment by jrgnicho** [comment]
@marrts ping

**Comment by marrts** [comment]
I was not planning on making additional changes at this time, is there anything you feel that needs to be addressed before this is merged in?

**Comment by jrgnicho** [comment]
Commits from this PR have already been merged

**Pull request closed without merging**